lib/tasks/gather.js

```diff
@@ -287,7 +287,10 @@ class TaskGather extends SttTask {
       this._startFinalAsrTimer();
       return;
     }
-    if (!resolved && this.interDigitTimeout > 0 && this.digitBuffer.length >= this.minDigits) {
+    if (!resolved &&
+        this.interDigitTimeout > 0 &&
+        this.digitBuffer.length >= this.minDigits &&
```

Comment:
I don't think the addition of the check of this.input should be necessary.  There should never be any digits in the buffer if we are not collecting digits.

Reply:
so how did digits get in the buffer?  That is what we should fix

Reply:
@xquanluu any thoughts on this?